validators/store/snapshot/snapshot.go

```diff
@@ -520,7 +520,7 @@ func (s *SnapshotValidatorStore) removeLowerSnapshots(
 	currentHeight uint64,
 ) {
 	// remove in-memory snapshots from two epochs before this one
-	lowerEpoch := int(currentHeight/s.epochSize) - 2
+	lowerEpoch := int64(currentHeight/s.epochSize) - 2
```

Comment:
Motivation for this change is not to limit ourselves only to `int64` range. WDYT?
Also this constant `preserveEpochs` should be placed in the const block at the top of the file.

```suggestion
	const preservedEpochs = 2

	currentEpoch := currentHeight / s.epochSize
	if currentEpoch < preservedEpochs {
		return
	}

	// remove in-memory snapshots from two epochs before this one
	lowerEpoch := currentEpoch - preservedEpochs
	purgeBlock := lowerEpoch * s.epochSize
	s.store.deleteLower(purgeBlock)
```

Reply:
Done